src/scvi/external/mrvi/_model.py

```diff
@@ -1112,7 +1112,7 @@ def differential_expression(
         """
         from functools import partial
 
-        from statsmodels.stats.multitest import multipletests
+        from scipy.stats import false_discovery_rate
```

Comment:
should be false_discovery_control, not rate. let's make sure this one passes tests.